tests/gridworld_test.py

```diff
@@ -279,14 +279,14 @@ def extract_coord_test():
         yield extract_coord_check, label, expected_coord
 
 
-def eq_GridWorld_check(G, H, eq):
```

Comment:
I agree with the name change. For posterity, the previous name was originally motivated to emphasize the relationship with the class `GridWorld` by preserving the letter casing of the class name.